url_test.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
```

Comment:
Type those value under `KEY` which you get by running `travis token --org` into your terminal.